setup.cfg

```diff
@@ -26,6 +26,7 @@ install_requires =
     django-machina==1.1
     oauthlib>=3.0.0
     django-haystack==3.0b1
+    draftjs_exporter
```

Comment:
Can you sort dependencies alphabetically please? This would ease project dependencies list maintenance.